### PR TITLE
WEB-1590: update changelog and refs for onboarding > buisness sales flow

### PR DESCRIPTION
**Pull request by cryptonym0**
Add required field Average Transaction Amount to PSP-CAD onboarding API workflow 
Add required field Average Time from Purchase to Delivery to PSP-CAD onboarding API workflow
Add required field Are You Seasonal to PSP-CAD onboarding API workflow

### CODE REVIEW

File: source/docs/guides/onboarding.md [hunk 1]
```diff
@@ -272,6 +272,9 @@ curl https://sandbox-api.na.bambora.com/v1/boarding/workflows/psp-cad/applicatio
     "business": {
       "company_name": "TEST_APPROVED",
       "annual_sales": 800000.00,
+      "average_amount": 150.00,
+      "average_delivery": "14",
+      "seasonal_sales": "1",
```
Comment (pzelnip) [inline source/docs/guides/onboarding.md]: Do we want this here?  `seasonal_sales` is different depending on if using the 2016 or 2017 version of the API (in 2016 it's `"1"` or `"0"`, in 2017 it's `true` or `false`), but I think the intention of this sample is to be a generic version-agnostic request.

File: source/docs/references/onboarding_API/index.md [hunk 2]
```diff
@@ -40,6 +40,15 @@ The following workflows are currently supported:
 
 [Terms API Spec](./tac_v2016-08-18)
 
+#### October 31st, 2019
+* New Fields for PSP-CAD and PSP-USA
+    * required `average_amount` : Average Transaction Amount `number` up to 10 digits
+    * required `average_delivery` : Average Time from Purchase to Delivery `string`
+    * required `seasonal sales` : Are You Seasonal `string`
```
Comment (pzelnip) [inline source/docs/references/onboarding_API/index.md]: Aren't these fields optional?

Comment (cryptonym0) [reply source/docs/references/onboarding_API/index.md]: Oui

File: source/docs/guides/onboarding.md (hunk 1 above)
Comment (cryptonym0) [reply source/docs/guides/onboarding.md]: roger

**Pull request merged**